### PR TITLE
Version bump for release

### PR DESCRIPTION
**Pull request by Goirad**

### CODE REVIEW

File: mbedtls/Cargo.toml [hunk 1]
```diff
@@ -33,7 +33,7 @@ rc2 = { version = "0.3", optional = true }
 rs-libc = "0.1.0"
 
 [dependencies.mbedtls-sys-auto]
-version = "2.18.0"
```
Comment (jethrogb) [inline mbedtls/Cargo.toml]: Unnecessary bump

Comment (Goirad) [reply mbedtls/Cargo.toml]: But the change we want to pick up is in mbedtls-sys-auto isn't it?

Comment (Goirad) [reply mbedtls/Cargo.toml]: Or do you mean that it goes from `2.18.0` to `2.18.2`?

Comment (jethrogb) [reply mbedtls/Cargo.toml]: If 2.18.0 is working fine for people then there's no reason to force them to upgrade.

Comment (jethrogb) [reply mbedtls/Cargo.toml]: So this means this file doesn't need to change at all.

Comment (Goirad) [reply mbedtls/Cargo.toml]: Understood

**Pull request merged**